docs/en/observability/whats-new.asciidoc

```diff
@@ -1,3 +1,5 @@
+:minor-version: 7.10
```

Comment:
I don't think this setting and unsetting is necessary. AFAIK, `:minor-version:` will always remain at 7.10. See 7.9's version file for proof: https://github.com/elastic/docs/blob/master/shared/versions/stack/7.9.asciidoc. Unsetting it at the end of this page would break its usage elsewhere in the Observability guide

Reply:
doh...face palm. Thanks for catching this Brandon.

Reply:
Thanks.